src/webview.cpp

```diff
@@ -1,53 +1,53 @@
 #include "webview.h"
-
 #include <QAction>
 #include <QWebEngineProfile>
 #include <iostream>
 #include "kiwixapp.h"
 #include "webpage.h"
 
 WebView::WebView(QWidget *parent)
-    : QWebEngineView(parent)
+    :QWebEngineView(parent)
 {
-    setPage(new WebPage(this));
+    setPage(new WebPage(this));   //to show the ZIM file as a webpage
```

Comment:
No.
We need a custom webpage to be able to open a external browser when the user click on a external link.
To do so, we need to overwrite `bool WebPage::acceptNavigationRequest` to check all request in the webView.
See https://github.com/kiwix/kiwix-desktop/blob/master/src/webpage.cpp